packages/docs/heading-docs/stories/heading.stories.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/no-unescaped-entities */
+import type { StoryObj } from '@storybook/react-vite';
+import type { HeadingProps } from '@nl-design-system-candidate/heading-react';
```

Comment:
```suggestion
import type { HeadingProps } from '@nl-design-system-candidate/heading-react/css?';
```

De CSS van `nl-heading` is niet included in de versie van de PR, komt dat hier door?